### PR TITLE
[1LP][RFR] Fixed failing tests according to new UI changes.

### PR DESCRIPTION
**Pull request by hhovsepy**
changed displaying domain name, add provider error message fix
{{pytest: cfme/tests/middleware/test_middleware_provider.py::test_duplicite_provider_creation -v --use-provider hawkular}}

### CODE REVIEW

File: cfme/middleware/domain.py [hunk 1]
```diff
@@ -124,8 +125,7 @@ def _domains_in_mgmt(cls, provider):
         rows = provider.mgmt.inventory.list_domain()
         for row in rows:
             domains.append(MiddlewareDomain(
-                name=row.data['Local Host Name']
-                if row.data['Local Host Name'] != 'master' else 'Unnamed Domain',
+                name=re.sub(r'master\.', '', re.sub(r'%20', ' ', row.path.feed_id)),
```
Comment (izapolsk) [inline cfme/middleware/domain.py]: OPTIONAL: I don't think you need re.sub here. You can do this with ordinary string's "replace" method.

File: cfme/tests/middleware/test_middleware_provider.py [hunk 2]
```diff
@@ -109,7 +109,8 @@ def test_required_fields_validation(provider, soft_assert):
 @pytest.mark.usefixtures('setup_provider')
 def test_duplicite_provider_creation(provider):
     """Tests that creation of already existing provider fails."""
-    with error.expected('Host Name has already been taken'):
+    with error.expected('Name has already been taken, '
```
Comment (izapolsk) [inline cfme/tests/middleware/test_middleware_provider.py]: REQUIRED: tests fails in 5.7 with flash message 
"Name has already been taken, Host Name has already been taken"

**Comment by hhovsepy** [comment]
Thanks for the review @izapolsk , have fixed the failing test.

**Comment by hhovsepy** [comment]
failing test passes

**Pull request merged**